examples/installation/dsf_single_account_deployment/variables.tf

```diff
@@ -448,6 +448,12 @@ variable "sonarw_gw_public_key_file_path" {
   description = "The Agentless Gateway sonarw user public key file path - used for remote Agentless Gateway federation, HADR, etc."
 }
 
+variable "sonar_machine_base_directory" {
```

Comment:
Nice, I agree that this variable should be added to all installation examples but not to poc

Reply:
Thank you